prow/plugins/assign/assign.go

```diff
@@ -134,13 +157,53 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 	if len(unassign) > 0 {
 		log.Printf("Removing assignees from %s/%s#%d: %v", ae.org, ae.repo, ae.number, unassign)
 		if err := gc.UnassignIssue(ae.org, ae.repo, ae.number, unassign); err != nil {
-			return err
+			msg := fmt.Sprintf("Unassign failed: %v.", err.Error())
```

Comment:
Do a type assertion on the error. See [Error handling and Go](https://blog.golang.org/error-handling-and-go) and [Type assertions](https://golang.org/ref/spec#Type_assertions).